src/Microsoft.IdentityModel.TestExtensions/TestTokenCreator.cs

```diff
@@ -288,9 +288,9 @@ public string CreateTokenWithMissingIssuedAt()
             var tokenDescriptor = CreateTokenDescriptorWithInstanceOverrides();
 
             // Remove the default value.
-            tokenDescriptor.Subject.RemoveClaim(
+            tokenDescriptor?.Subject?.RemoveClaim(
                 tokenDescriptor.Subject.FindFirst(JwtRegisteredClaimNames.Iat));
-            return CreateToken(tokenDescriptor);
+            return CreateToken(tokenDescriptor!);
```

Comment:
why do we need to say this is not null here but not at the other call sites for CreateToken?